csv2md/csv2md.py

```diff
@@ -37,13 +50,37 @@ def main():
                         help='delimiter character. Default is \',\'')
     parser.add_argument('-q', '--quotechar', metavar='QUOTECHAR', type=str, default='"',
                         help='quotation character. Default is \'"\'')
+    parser.add_argument('--center-fields', metavar='CENTER_FIELDS', type=str, default='',
+                        help='fields to center aligned')
+    parser.add_argument('--right-fields', metavar='RIGHT_FIELDS', type=str, default='',
+                        help='fields to right aligned')
     args = parser.parse_args()
+    
+    center_fields = set()
+    if args.center_fields:
+        center_fields = set((int(f) for f in args.center_fields.split(',')))
+    right_fields = set()
+    if args.right_fields:
+        right_fields = set((int(f) for f in args.right_fields.split(',')))
```

Comment:
Instead of custom parsing use [nargs](https://docs.python.org/3/library/argparse.html#nargs) with `*` parameter. Something like this:

```python
parser.add_argument('--center-aligned-columns', metavar='CENTER_ALIGNED_COLUMNS',
                    nargs='*', type=int, default=[],
                    help='column numbers with center alignment')
```